src/js/constants.js

```diff
@@ -791,6 +793,14 @@ export const EXPLORE_HEADER_ARTICLES_LABELS = {
     label: "Title",
     info: ""
   },
+  "supplements.financial_disclosures__asap": {
```

Comment:
@JosephMcArthur Could you tell me where we can see these? 

Another helpful thing to do in PRs would be to just screenshot the before/after of your changes so I know exactly what I should be looking out for

Reply:
`main` was just merged into this branch so that it can benefit from the latest fixes :)